cmd/aries-agentd/main_test.go

```diff
@@ -103,7 +103,7 @@ func validateRequests(t *testing.T) {
 								"@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/didexchange/1.0/invitation"
 							}`),
 				"application/didcomm-envelope-enc"),
-			expectedStatus:     http.StatusAccepted,
+			expectedStatus:     http.StatusInternalServerError,
```

Comment:
why this change?

Reply:
[Here](https://github.com/hyperledger/aries-framework-go/pull/240#discussion_r325327735).